### ISSUE TITLE
Saliency map for a chosen sample with its features.

### ISSUE THREAD

**Issue by vinayakumarr** [fchollet/keras#6043]
K. Simonyan, A. Vedaldi, and A. Zisserman, “Deep Inside Convolutional Networks: Visualising Image Classification Models and Saliency Maps,” arXiv preprint arXiv:1312.6034, 2013

Hi, 
      I have trained a CNN model and I want to create a Saliency map for a chosen sample with its features. This is plotted by using Taylor expansion and compute first partial derivative of the classification results before putting them through the softmax function. Does anybody have a sample code to do this?

**Comment by AvantiShri** [comment]
See this thread: https://github.com/fchollet/keras/issues/1777
